frontend/src/components/App/Layout.tsx

```diff
@@ -44,11 +54,133 @@ const useStyle = makeStyles(theme => ({
 
 export interface LayoutProps {}
 
+const CLUSTER_FETCH_INTERVAL = 10 * 1000; // ms
+
+interface Config {
+  [prop: string]: any;
+}
+
+function ClusterNotFoundPopup(props: {
+  isClusterNotAvailable: boolean;
+  changeClusterNotAvailableStateHandler: (isClusterNotAvailable: boolean) => void;
+}) {
+  const { isClusterNotAvailable, changeClusterNotAvailableStateHandler } = props;
+  const theme = useTheme();
+  const cluster = getCluster();
+  const { t } = useTranslation('frequent');
+  const isSidebarOpen = useTypedSelector(state => state.ui.sidebar.isSidebarOpen);
+  const history = useHistory();
+
+  if (!cluster) {
+    return null;
+  }
+
+  if (!isClusterNotAvailable) {
+    return null;
+  }
+  return (
+    <>
```

Comment:
AFAIU, you don't need the fragment here since you only have one element (Box) inside it.